src/navx_node.cpp

```diff
@@ -2,25 +2,50 @@
  * navx_node.cpp
  * Runs the Kauai Labs NavX, using modified NavX library
  * VERSION: 1.0.0
- * Last changed: 2019-10-05
+ * Last changed: 2019-10-31
  * Authors: Jude Sauve <sauve031@umn.edu>
  * Maintainers: Nick Schatz <schat127@umn.edu>
  * MIT License
  * Copyright (c) 2019 UMN Robotics
  */
 
+/* Interface: 
+ *  Pub: 
+ *   imu/data (sensor_msgs/Imu)
+ *   imu/euler (geometry_msgs/Point)
```

Comment:
You do want to specify a name for these publishers, because the topics are not always the same as what you intend it to be, and the types are often too generic. 
By 'name', I mean that the Publisher/Subscriber object, any boolean param to turn on/off, the callback, and any other related data should keep the same prefix. 
Ex: euler_pub, publish_euler, (not in this) euler_sub, euler_callback, euler_msg, euler_...
This also is important for when there are multiple pubs/subs of same type, in very nearby topics. 
For example, the map-global ekf would listen to "odometry/filtered" to publish an "odometry/filtered_map"
Example:
```
// Pub
//    imu_pub (sensor_msgs/Imu): "imu/data"
//    euler_pub (geometry_msgs/Point): "imu/euler"

// Sub
//    magnetometer_sub (...): "..."
```